packages/next/build/index.ts

```diff
@@ -851,6 +853,11 @@ export default async function build(dir: string, conf = null): Promise<void> {
       JSON.stringify(prerenderManifest),
       'utf8'
     )
+    await generateClientSsgManifest(prerenderManifest, {
```

Comment:
Why can't this be part of the buildManifest?

Reply:
I intentionally separated them so we didn’t unnecessarily increase the size of either file.

Also, their generation is different. The build manifest can be computed at build time but the SSG manifest must be post build after we require all the pages in `jest-worker`.